sdk/synapse/azure-synapse-artifacts/CHANGELOG.md

```diff
@@ -1,14 +1,197 @@
 # Release History
 
-## 0.19.1 (Unreleased)
+## 0.20.0 (2025-03-07)
 
 ### Features Added
 
-### Breaking Changes
+  - Add a new Model IcebergDataset
+  - Add a new Model IcebergSink
+  - Add a new Model TeradataSink
+  - Model AzureFileStorageLinkedService has a new parameter service_endpoint
```

Comment:
https://github.com/Azure/azure-sdk-for-python/tree/6f44195acc3ef790bcc7f6b3cfbeb925e4b05baa/sdk/synapse/azure-synapse-artifacts

Since python 3.7 has been dropped support, you shall also update README.md from `Python 3.7+` to `Python 3.8+`